src/Pate/Verification.hs

```diff
@@ -110,37 +119,44 @@ verifyPairs logAction elf elf' blockMap dcfg pPairs = do
     Nothing -> throwError $ equivalenceError UnsupportedArchitecture
     Just vs -> pure vs
   ha <- liftIO CFH.newHandleAllocator
-  (mainO, pfmO)  <- runDiscovery elf
-  (mainP, pfmP) <- runDiscovery elf'
+  (oMain, oPfm)  <- runDiscovery elf
+  (pMain, pPfm) <- runDiscovery elf'
 
-  liftIO $ LJ.writeLog logAction (PE.LoadedBinaries (elf, pfmO) (elf', pfmP))
+  liftIO $ LJ.writeLog logAction (PE.LoadedBinaries (elf, oPfm) (elf', pPfm))
 
   Some gen' <- liftIO N.newIONonceGenerator
-  let pfeats = W4PF.useBitvectors .|. W4PF.useSymbolicArrays
+  let pfeats = W4PF.useBitvectors .|. W4PF.useSymbolicArrays .|. W4PF.useIntegerArithmetic .|. W4PF.useStructs
   CBO.withYicesOnlineBackend W4B.FloatRealRepr gen' CBO.NoUnsatFeatures pfeats $ \sym -> do
```

Comment:
I don't recall - do we take advantage of the online backend? Would it simplify anything to use the offline backend? We don't have to do it now, but knowing would be useful (e.g., we might be able to parallelize)

Reply:
I added a comment about this. It's definitely possible but not completely trivial to swap to an offline backend.